antarest/study/web/variant_blueprint.py

```diff
@@ -421,4 +422,32 @@ def create_from_variant(
         params = RequestParameters(user=current_user)
         raise NotImplementedError()
 
+    @bp.put(
+        "/studies/variants/clear-snapshots",
+        tags=[APITag.study_variant_management],
+        summary="Clear variant snapshots",
+        responses={
+            200: {
+                "description": "Delete snapshots older than a specific number of hours. By default, this number is 24."
+            }
+        },
+    )
+    def clear_variant_snapshots(
```

Comment:
After more thinking:
it could be relevant to execute this cleanup as a `Task` by the task service, because it will take a long time to execute.

Then here we should return the task ID. You can for example look at the `THERMAL_CLUSTER_SERIES_GENERATION` that Martin added recently, to see how it's done! it's quite simple you need to add a new task type and provide the function to be executed to the task service, instead of executing it directly in the variant service.